pkg/schedule/operator/operator_controller.go

```diff
@@ -329,6 +329,7 @@ func (oc *Controller) AddOperator(ops ...*Operator) bool {
 	// but maybe user want to add operator when waiting queue is busy
 	if oc.exceedStoreLimitLocked(ops...) {
 		for _, op := range ops {
+            operatorCounter.WithLabelValues(op.Desc(), "store_limit").Inc()
```

Comment:
It cloud be same as Line366/367
```suggestion
            operatorCounter.WithLabelValues(op.Desc(), "exceed-limit").Inc()
```